app/helpers/competition_api.rb

```diff
@@ -122,6 +122,18 @@ def user_can_cancel?
     @competition_json['allow_registration_self_delete_after_acceptance']
   end
 
+  def qualifications
+    @competition_json['qualifications']
+  end
+
+  def get_qualification_for(event)
+    @competition_json['qualifications'][event]
+  end
+
+  def enforces_qualifications?
+    @competition_json['qualification_results'] == 1 && @competition_json['allow_registration_without_qualification'] == 0
```

Comment:
I'm a bit confused why you use 1 and o instead of true or false?

Reply:
Was looking at the database not the JSON output - fixed